streamz/sources.py

```diff
@@ -71,3 +72,42 @@ def start(self):
                     yield last
                 else:
                     return
+
+
+class Kafka(Source):
+    """ Accepts messages from Kafka
+
+    Parameters
+    ----------
+    topics: list of str
+        Labels of Kafka topics to consume from
+    url: str
+        Connection string (host:port) by which to reach Kafka
+    group: str
+        Identity of the consumer. If multiple sources share the same group,
+        each message will be passed to only one of them.
+    poll_interval: number
+        Seconds that the thread sleeps between polling Kafka for new messages
+    """
+    def __init__(self, topics, url='localhost:9092', group='streamz',
+                 poll_interval=0.1):
+        import confluent_kafka as ck
+        IOLoop.current().add_callback(self.poll_kafka)
```

Comment:
Given that this explicitly interacts with an IOLoop, is it a good idea to allow the user to pass in their instance?

Reply:
Yes, I think so.  In another branch I'm doing `loop = loop or IOLoop.current()`